usr/sbin/p11sak/p11sak.c

```diff
@@ -931,12 +931,10 @@ static CK_RV set_battr(const char *attr_string, CK_ATTRIBUTE *attr,
         for (i = 0; i < (int) strlen(attr_string); i++) {
             // attr_string length is checked in parse_gen_key_args to avoid memory problems
             if (prv == 1 && attr_na(char2attrtype(toupper(attr_string[i])), kt_PRIVATE) == 0) {
-                printf("private: %d\n", attr_string[i]);
```

Comment:
There was a comment in the commit I did (if i remember correctly), suggesting to leave these logs in there, to tell the user, which attributes (of the users input attributes) are actually used for the key creation and which might have been skipped, because of whatever reason.

Reply:
I don't think that the user understands the output as printed. Its internal information. If really required, then we need to print this in a more user friendly way. I would say lets remove it for now, and if it turns out that this information is really needed, then implement it in a user friendly way.